addon/webextension/background/main.js

```diff
@@ -77,6 +81,26 @@ window.main = (function () {
         .then(() => sendEvent("start-shot", "context-menu")));
   }));
 
+  function urlEnabled(url) {
+    if (opensMyShots(url)) {
+      return true;
+    }
+    if (url.startsWith(backend) || /^(?:about|data|moz-extension):/i.test(url)) {
```

Comment:
There might be a whitelist of WebExtension schemes, but I don't know what it is exactly